frontend/src/pages/ChoosePodProviderPage/ChoosePodProviderPage.tsx

```diff
@@ -0,0 +1,225 @@
+import React, { useEffect, useState } from 'react';
+import { useTranslate } from 'react-admin';
+import {
+  List,
+  ListItemButton,
+  ListItemText,
+  ListItemAvatar,
+  Avatar,
+  Divider,
+  Button,
+  TextField,
+  Grid
+} from '@mui/material';
+import AddIcon from '@mui/icons-material/Add';
+import StorageIcon from '@mui/icons-material/Storage';
+import { validateBaseUrl } from '../../utils';
+import SimpleBox from '../../layout/SimpleBox';
+
+const fetchRemotePodProviders = async (source: string) => {
+  const results = await fetch(source, {
+    headers: {
+      Accept: 'application/ld+json'
+    }
+  });
```

Comment:
The issue I'm facing is that I wanted provider hosters to have the opportunity to specify their own resources of which providers to recommend somewhere.
This doesn't seem to be possible with the data provider config, since you need to specify the `baseUrl` in advance. Or is there a way around somehow?

Reply:
If someone is not happy with the list provided in https://data.activitypods.org/pod-providers, they can create their own server and change the baseUrl. This could be an ENV var. In the base configuration, TrustedApp and PodProvider resources would be tied to the same server, but this could also be changed in a custom configuration. Well, basically anything is possible with the combination of dataServers/dataModels :-)

Reply:
Okay, so I'll create an ENV variable like ACTIVITYPODS_COMMON_CONF_URL which by default links to https://data.activitypods.org?